### PR TITLE
completed demo page styling 

### PR DESCRIPTION
**Pull request by Lujaina-Chata**
Completed edits. Waiting for review

### CODE REVIEW

File: example/src/SentimentAnalysisPage.js [hunk 1]
```diff
@@ -282,7 +287,7 @@ export default class SentimentAnalysisPage extends React.Component {
       <div style={{ padding: '20px', display: 'flex' }}>
         <div style={{ flex: 1 }}>
           <h2>Leave a Review</h2>
-          <Rating
+          {/* <Rating
```
Comment (Ben-CHATA) [inline example/src/SentimentAnalysisPage.js:290]: remove commented code

File: example/src/SentimentAnalysisPage.js [hunk 2]
```diff
@@ -293,7 +298,36 @@ export default class SentimentAnalysisPage extends React.Component {
                 rating: newRating.toFixed(1),
               })
             }
-          />
+          /> */}
+
+            {<Rating 
+              //defaultValue={2}
```
Comment (Ben-CHATA) [inline example/src/SentimentAnalysisPage.js:304]: remove

File: example/src/SentimentAnalysisPage.js [hunk 3]
```diff
@@ -293,7 +298,36 @@ export default class SentimentAnalysisPage extends React.Component {
...
+              value={this.state.rating} 
+
+              onChange={(event, newRating) =>
+                //this.setState({
+                //  rating: newRating.toFixed(1),
+                //})
+                console.log(event)
+              }
```
Comment (Ben-CHATA) [inline example/src/SentimentAnalysisPage.js:312]: remove this whole onChange prop. it's not doing anything useful. We also generally don't want console logs in deployed environments.

File: example/src/SentimentAnalysisPage.js [hunk 4]
```diff
@@ -293,7 +298,36 @@ export default class SentimentAnalysisPage extends React.Component {
...
+          } } 
+        <input type="Clear" id="inputname" className="form-control" ref={el => this.inputTitle = el} />   
+        <textarea id="inputage" ref={el => this.inputEntry = el} className="form-control" />
+        <button className="btn btn-info" onClick={this.sendThru}>Add</button>
+
+        sendThru() {
+        //{this.inputTitle.value = "",
+        //this.inputEntry.value = "",}
+      }  
+      ref={el => this.inputTitle = el} */}
```
Comment (Ben-CHATA) [inline example/src/SentimentAnalysisPage.js:328]: remove all of this

**Review by Ben-CHATA — DISMISSED** [review]

**Review by Ben-CHATA — APPROVED** [review]

**Comment by Lujaina-Chata** [comment]
feat(restyling): admin control panel, toggle switch + V2.1 model, control signal input, half star + clearing, restyling and cleaning up spacing and headers.

**Pull request merged**